client/src/components/FilterTable/FilterTable.stories.js

```diff
@@ -0,0 +1,164 @@
+import _ from "lodash";
+import React from "react";
+
+import { create_full_results_hierarchy } from "src/panels/panel_declarations/results/result_drilldown/result_hierarchies";
+
+import { text_maker } from "src/panels/panel_declarations/results/result_text_provider";
+
+import {
+  status_key_to_glossary_key,
+  ordered_status_keys,
+  result_color_scale,
+} from "src/panels/panel_declarations/results/results_common";
+
+import { toggle_list } from "src/general_utils";
+import {
+  IconCheck,
+  IconAttention,
+  IconNotApplicable,
+  IconClock,
+} from "src/icons/icons";
+
+import { FilterTable } from "./FilterTable";
+
+export default {
+  title: "Tables/FilterTable",
+  component: FilterTable,
+};
+
+const Template = (args) => <FilterTable {...args} />;
+
+const active_list = [];
```

Comment:
Delete lines 31 to 156, just hard code some counts with inlined svgs. The click callback will need to control some state in `Template` to be effective for selecting items on click, putting in `args` won't work.

Reply:
Done!